tests/dragonfly/snapshot_test.py

```diff
@@ -254,13 +254,54 @@ class TestDflySnapshotOnShutdown(SnapshotTestBase):
     def setup(self, tmp_dir: Path):
         self.tmp_dir = tmp_dir
 
+    async def _get_info_memory_fields(self, client):
+        res = await client.execute_command("INFO MEMORY")
+        fields = {}
+        for line in res.decode("ascii").splitlines():
+            if line.startswith("#"):
+                continue
+            k, v = line.split(":")
+            if k == "object_used_memory" or k.startswith("type_used_memory_"):
+                fields.update({k: int(v)})
+        return fields
+
+    async def _delete_all_keys(self, client):
+        # Delete all keys from all DBs
+        for i in range(0, SEEDER_ARGS["dbcount"]):
+            await client.select(i)
+            while True:
+                keys = await client.keys("*")
+                if len(keys) == 0:
+                    break
+                await client.delete(*keys)
+
+    @pytest.mark.asyncio
+    async def test_memory_counters(self, df_seeder_factory, df_server):
+        a_client = aioredis.Redis(port=df_server.port)
+
+        memory_counters = await self._get_info_memory_fields(a_client)
+        assert memory_counters == {"object_used_memory": 0}
+
+        seeder = df_seeder_factory.create(port=df_server.port, **SEEDER_ARGS)
+        await seeder.run(target_deviation=0.1)
+
+        memory_counters = await self._get_info_memory_fields(a_client)
+        assert memory_counters["object_used_memory"] > 0
```

Comment:
isnt this check is included in the line below?

Reply:
yes it is :facepalm: